src/Components/IndividualArticle.jsx

```diff
@@ -0,0 +1,26 @@
+import IndividualArticleCard from "./IndividualArticleCard";
+import {fetchArticleById} from '../utils/api-utils';
+import { useEffect, useState } from "react";
+import { useParams } from "react-router";
+
+
+export default function IndividualArticle() {
+    const {article_id} = useParams()
+    const [ article, setArticle] = useState([])
+
```

Comment:
Please set up a state for loading and display a message to the user while the article is loading.

Reply:
added loading to both all articles and individual article